tests/sample_size/test_metrics.py

```diff
@@ -118,56 +119,60 @@ def test_boolean_metric_get_probability_too_small(self):
             "Error: Please provide a float between 0 and 1 for probability.",
         )
 
-    @parameterized.expand(product((1, 2, 10), (2, 10)))
+    @parameterized.expand(product((1, 2, 10), (2, 10), TEST_ALTERNATIVES))
```

Comment:
> Approving, though having a test under `tests_metrics.py` that tests the cartesian product of alternatives and metric types might be prudent

@vjohnson1-godaddy Do you mean having a new test function in `tests_metrics.py` to create a new metric and calculate its p-values, where the test cases are the cartesian product of alternatives and metric types? It will overlap with `test__generate_alt_p_values` in each metric type (or do you mean removing these `test__generate_alt_p_values` and instead using one function to test all possible metric type - alternative combinations?)

Reply:
Yep, the latter! I think we can summarize it all with just the one test. But not a blocker